deploy/crds/operator.ibm.com_v1alpha1_metaoperatorconfig_cr.yaml

```diff
@@ -13,3 +13,16 @@ spec:
     spec:
       etcdCluster:
         size: 1
+  - name: ibm-licensing-operator
+    spec:
+      IBMLicensing:
+        apiSecretToken: ibm-licensing-token
+        datasource: datacollector
+        httpsCertsSource: self-signed
+        httpsEnable: false
+        imageRegistry: hyc-cloud-private-integration-docker-local.artifactory.swg-devops.com/ibmcom/ibm-licensing-amd64
+        imageTagPostfix: 1.0.0
+        logLevel: INFO
+        instanceNamespace: kube-system
```

Comment:
the imageRegistry and this namespace can be moved to the default value of the CSV file.

Later when you want to change something, you don't need to update meta-operator anymore.

You just need to update license operator and things will be taken effect.

https://github.com/IBM/ibm-licensing-operator/blob/create_csv/deploy/olm-catalog/ibm-licensing-operator/1.0.0/ibm-licensing-operator.v1.0.0.clusterserviceversion.yaml#L13-L21